tests/sys_crypto/tests-crypto-modes-ccm.c

```diff
@@ -1009,6 +1009,8 @@ static void test_crypto_modes_ccm_check_len(void)
 {
     int ret;
 
+/* Using length_encoding above UINT16_MAX doesn't work on 8/16 bit architectures */
+#if SIZE_MAX < UINT16_MAX
```

Comment:
This `#if` statement doesn't make sense, I think the comparison is reversed, the test should be enabled when `SIZE_MAX` is bigger than `UINT16_MAX`

Reply:
Oops... Fixed but now it raises another issue with a condition that is always false on 8/16 bit architecture => c159dc2.